code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -928,6 +928,14 @@
 	taste_description = "milky batter"
 	color = "#fccc98"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+	
+/datum/reagent/consumable/unfriedegg // Made with salt and pepper to keep the white & yolk seperate till needed
+	name = "Unfried Egg"
```

Comment:
just call it cracked egg or whatever what the fuck

Reply:
People will ironically throw this into the deepfryer

Reply:
But it's not a cracked egg. It's also not plain raw egg